### ISSUE TITLE
Accessibility - Individual Country Screen - Talkback Does Not Notifying User Goods Icons are Can be Tapped

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#35]
TalkBack does not notify the user that the different goods are a link/button to another screen/content in the application.

**Comment by aquilent-dgemoets** [comment]
Android TalkBack does not automatically announce/identify links/buttons as selectable; this must be added explicitly by the developer.  We will add heading indicators (in "alt tags") in as many places as time permits (also applies to #46 and #51).

**Comment by jfc3-dol** [comment]
It would be best to add that a button or link is tap-able or clickable to TalkBack users know there is more data/information on another screen similar to what was done on the home screen for ILAB and DOL websites at the bottom of the screen.

**Comment by aquilent-dgemoets** [comment]
Agreed-- we actually looked into it last night but it was non-trivial to fix due to the compound nature of the Goods icon (good name plus 1-3 exploitation types).  We still hope to address this in the final release.

**Comment by nmangu** [comment]
The latest BETA-5 release reads the Goods Icon name and CL/FL. When double tapped it goes into the detail page, however does not read the whole content of the page. It just reads the main heading, e.g., Bidis (hand-rolled cigarettes)  and the text 'Produced with Exploitive Labor in 2 countries'.

**Comment by jfc3-dol** [comment]
TalkBack was able to read all the exploited goods I checked as a button and that you can double tap to activate them.
